### PR TITLE
Fixed some documentation spelling mistakes

### PR DESCRIPTION
**Pull request by ErfanTagh**
## What changes are proposed in this pull request?

(Please fill in changes proposed in this fix)

## How is this patch tested? If it is not, please explain why.

(Details)

### Is this a user-facing change that should be mentioned in the release notes?

-   [ ] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [ ] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

File: docs/source/user_guide/plots.rst [hunk 1]
```diff
@@ -860,7 +860,7 @@ Plotting backend
 ----------------
 
 Most plotting methods in the
-:meth:`fiftyone.core.plots <fiftyone.core.plots.base>` module provide an
+:meth:`fiftyone.core.plots <fiftyone.core.plots.base>` module provides an
```
Comment (brimoor) [inline docs/source/user_guide/plots.rst:863]: `provide` is actually correct here

**Review by brimoor — CHANGES_REQUESTED** [review]
Thanks @ErfanTagh for fixing these mistakes! There's one change that needs to be reversed but then this is good to go 💯

File: docs/source/user_guide/plots.rst (hunk 1 above)
Comment (ErfanTagh) [reply docs/source/user_guide/plots.rst]: Thanks, Brian, I reverted it back to the original now

**Review by brimoor — APPROVED** [review]
LGTM

**Pull request merged**